app/helpers/range-slider.js

```diff
@@ -0,0 +1,3 @@
+import nouislider from 'ember-cli-nouislider';
+
+export default nouislider;
```

Comment:
add empty line.

Reply:
why is this a helper, shouldn't it be a component?

Reply:
I added this as a helper as I saw the error similar to `no helper found` when I didn't had it. Then I saw many other helpers defined too. I'll add it as a component. I guess it will be more suited as a component.

Reply:
@CosmicCoder96 I added this as a component in the root of the components,
it tries to import the module nouislider from `open-event-frontend/components/range-slider` and not from the `node_modules`.

Reply:
just follow the docs of the addon, they clearly say to generate a component.